models/models/billing.py

```diff
@@ -208,3 +219,57 @@ def from_json(record):
             cost=record.get('cost'),
             currency=record.get('currency'),
         )
+
+
+class BillingCostDetailsRecord(SMBase):
+    """_summary_"""
+
+    cost_group: str
+    cost_category: str
+    daily_cost: Decimal | None
+    monthly_cost: Decimal | None
```

Comment:
Is there a reason for using Decimal over float?

Reply:
Nope, I've changed to float.